### DISCUSSION TITLE
Extension Support for Visual Studio 2022

### DISCUSSION THREAD

**Discussion by revolutionxk** [community/community#8927] [Copilot Conversations] [answered]
I currently use Visual Studio 2022 frequently for my work, I have the option to use CLion/Rider or Visual Studio Code for C++ and C#, but currently the Visual Studio 2022 compiler, performance and auto-completion is better than other platforms and it's not feasible for me to switch to a supported platform to just use Copilot, I really like Copilot it's an amazing tool and it would help me a lot in my work if I had support for this platform.

**Comment by nepton** [comment]
Github Copilot is an amazing tool that can help me with my daily work. The visual studio 2022 is one of most popular IDE. I am looking forward to this extension  :)

**Comment by LyeZinho** [comment]
Copilot could be added for visual stuidio in nexts vs updates when copilot is open for public
this will help a lot of my projects

**Comment by OctoSheep** [comment, marked as answer]
[Here](https://marketplace.visualstudio.com/items?itemName=GitHub.copilotvs) it is

**Reply by LyeZinho** [reply]
thanks

**Reply by revolutionxk** [reply]
Thank you
